### ISSUE TITLE
`git clone core repo` is not idempotent due to LF normalization

### ISSUE THREAD

**Issue by Yonezawa-T2** [personium/ansible#21]
`git clone core repo` in `git_clone.yml` is not idempotent because some files have CRLF line endings while decleared `text eol=lf` in `.gitattributes`, so that those files are regarded as “modified” just after cloned, and `git` in Ansible refuse to override modifications unless `force: yes`.

```
[/tmp/personium-core]$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   src/main/resources/ajax.js
	modified:   src/main/resources/html/authform.html
	modified:   src/main/resources/html/error.html
```

We should fix those line endings, or have `force: yes` in `git_clone.yml`.

**Comment by hiroaki-shibata** [comment]
Thank you for your comment. We will check line endings and fix them.
